python/tests/test_oberon.py

```diff
@@ -13,7 +14,7 @@
 
 class OberonTests(unittest.TestCase):
     def test_oberon_demo(self):
-        key = oberon.create_key(CreateOberonKeyRequest())
+        key = oberon.create_key(CreateOberonKeyRequest(seed=bytes([1, 2, 3])))
```

Comment:
Should this be in a separate PR?

Reply:
Or removed, was diagnosing something in dart.